src/ebay-listbox/listbox-option.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ComponentProps, FC, ReactElement } from 'react'
+import classNames from 'classnames'
+import { filterByType } from '../utils'
+import { EbayIcon } from '../ebay-icon'
+import { EbayListboxOptionDescription } from './listbox-option-description'
+
+export type EbayListboxOptionProps = ComponentProps<'div'> & {
+    icon?: ReactElement;
+    text?: string;
+    value: string;
+    disabled?: boolean;
+    selected?: boolean;
+};
+
+export const EbayListboxOption: FC<EbayListboxOptionProps> = ({
+    className,
+    icon,
+    text,
+    children,
+    disabled,
+    tabIndex,
+    selected,
+    ...rest
+}) => {
+    const description = filterByType(children, EbayListboxOptionDescription)
+    const displayText = text || (!description?.length ? children : '')
+
+    return (
+        <div
+            {...rest}
+            tabIndex={disabled ? -1 : tabIndex}
```

Comment:
is it ok to skip for a11y as well when its disabled?

Reply:
Yes so it cannot be accessed with the tab index